jenkins-integration/jenkins-jobs/common/scripts/job-steps/initialize_ruby_env.sh

```diff
@@ -4,7 +4,7 @@ echo ""
 echo "Setting Up Ruby Environment"
 echo ""
 source /usr/local/rvm/scripts/rvm
-rvm use 2.2.5 || exit 1
```

Comment:
Is the version of ruby important? Do we just need 2.x? Just wondering if we could less tightly couple this script from the contents of puppetlabs-puppetserver_perf_driver? (or if that's even desirable)

Reply:
I don't think the version of Ruby matters particularly. We could update the driver module to set the version it installs as `default` and then switch to that here, to avoid the coupling, but I do know in all of our Jenkins infrastructure in general, we tend to be specific about which version we are using right in the script. Not sure if that's good or bad.

Reply:
I think it's a good thing in CI because there's (was?) multiple rubies installed to support both teams that needed to run spec tests against multiple rubies as well as projects that needed to run different versions of beaker that used different versions of ruby (eg. puppetserver 1.x and 2.x branches used different major versions of beaker, one requiring 1.9 and the other able to use ruby 2.x).

Unless we're going to be supporting multiple rubies I'd prefer not having to make the change in two places when we want to bump the ruby version.

Reply:
So the RVM default on the real perf driver is 1.9.3. Not sure if we can change that. So we may still have to set the version explicitly here (to something other than 2.4.2, since that version is not installed), unless we change that about the real perf driver.